internal/kafka/internal/clusters/ocm_provider.go

```diff
@@ -139,66 +139,6 @@ func (o *OCMProvider) ApplyResources(clusterSpec *types.ClusterSpec, resources t
 	return &resources, nil
 }
 
-func (o *OCMProvider) ScaleUp(clusterSpec *types.ClusterSpec, increment int) (*types.ClusterSpec, error) {
-	_, err := o.ocmClient.ScaleUpComputeNodes(clusterSpec.InternalID, increment)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to scale up cluster %s by %d nodes", clusterSpec.InternalID, increment)
-	}
-	return clusterSpec, nil
-}
-
-func (o *OCMProvider) ScaleDown(clusterSpec *types.ClusterSpec, decrement int) (*types.ClusterSpec, error) {
-	_, err := o.ocmClient.ScaleDownComputeNodes(clusterSpec.InternalID, decrement)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to scale down cluster %s by %d nodes", clusterSpec.InternalID, decrement)
-	}
-	return clusterSpec, nil
-}
-
-func (o *OCMProvider) SetComputeNodes(clusterSpec *types.ClusterSpec, numNodes int) (*types.ClusterSpec, error) {
-	_, err := o.ocmClient.SetComputeNodes(clusterSpec.InternalID, numNodes)
-	if err != nil {
-		return nil, errors.Wrapf(err, "failed to set compute nodes value of cluster %s to %d", clusterSpec.InternalID, numNodes)
-	}
-	return clusterSpec, nil
-}
-
-func (o *OCMProvider) GetComputeNodes(clusterSpec *types.ClusterSpec) (*types.ComputeNodesInfo, error) {
```

Comment:
To keep in mind, this might be useful in the near future

Reply:
I do not see how this will be used.

Reply:
To potentially perform some kind of comparison of desired vs existing nodes